src/views/NotFound.vue

```diff
@@ -20,5 +20,8 @@ import GenericError from 'theme/components/GenericError.vue';
 export default {
   name: 'NotFound',
   components: { GenericError },
+  created() {
+    this.$emit('availableLocales', []);
```

Comment:
Hm. Wouldn't we want the full range of supported "app" locales for pages like this which have no user content but still can have localized app-produced error messages?

Reply:
True. I decided that these pages shouldn't change the previous availableLocales, so I deleted this change